packages/core/src/services/documents/update.ts

```diff
@@ -1,51 +1,56 @@
 import { omit } from 'lodash-es'
 
+import { database } from '$core/client'
 import { findWorkspaceFromCommit } from '$core/data-access'
 import { Result, Transaction, TypedResult } from '$core/lib'
 import { BadRequestError, NotFoundError } from '$core/lib/errors'
 import { DocumentVersionsRepository } from '$core/repositories'
 import { Commit, DocumentVersion, documentVersions } from '$core/schema'
+import { assertCommitIsDraft } from '$core/services/documents/utils'
 import { eq } from 'drizzle-orm'
 
 // TODO: refactor, can be simplified
-export async function updateDocument({
-  commit,
-  document,
-  path,
-  content,
-  deletedAt,
-}: {
-  commit: Commit
-  document: DocumentVersion
-  path?: string
-  content?: string | null
-  deletedAt?: Date | null
```

Comment:
Removed `deletedAt` from here now that we specific services for deleting documents.